pubnub-gson/src/main/java/com/pubnub/internal/v2/callbacks/DelegatingEventListener.java

```diff
@@ -0,0 +1,207 @@
+package com.pubnub.internal.v2.callbacks;
```

Comment:
Why this folder is called "v2"?

Reply:
it mirrors the com.pubnub.api.* packages and contains their internal counterparts

Reply:
Ok. Thanks.